apps/omg_api/test/support/integration/deposit_helper.ex

```diff
@@ -25,10 +25,14 @@ defmodule OMG.API.Integration.DepositHelper do
   def deposit_to_child_chain(to, value, token \\ @eth)
 
   def deposit_to_child_chain(to, value, @eth) do
-    {:ok, receipt} = Eth.RootChain.deposit(value, to) |> Eth.DevHelpers.transact_sync!()
+    {:ok, %{"blockNumber" => deposit_eth_height} = receipt} =
```

Comment:
Shouldn't we refactor this function and the latter one (version for ERC20 token)? Besides the preparation in the latter one, they are basically the same. They just call different Root Chain functions, but checks are the same.